pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentTopicTest.java

```diff
@@ -558,10 +559,18 @@ public void testCreateTopicWithZombieReplicatorCursor(boolean topicLevelPolicy)
         admin.tenants().updateTenant("prop", tenantInfo);
 
         if (topicLevelPolicy) {
-            admin.topics().setReplicationClusters(topicName, Collections.singletonList(remoteCluster));
+            // setReplicationClusters may fail, so do retry.
+            Awaitility.await().until(() -> {
+                try {
+                    admin.topics().setReplicationClusters(topicName, Arrays.asList(remoteCluster, "test"));
+                    return true;
+                } catch (Exception ex) {
+                    return false;
+                }
```

Comment:
You could use `Awaitility.await().ignoreExceptions().untilAsserted(` instead

Reply:
Good idea. Thanks, already change to `Awaitility.await().ignoreExceptions().untilAsserted`